notebooks/01_collect_data/raw_to_text.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
<p>Before the saving cell here can you add a small section comparing one or two outputs from the original data to the new data to provide someone a view on how the data has been cleaned and improved. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/mailing-list-analysis-toolkit/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>